### PR TITLE
Feature/user registration

### PR DESCRIPTION
**Pull request by deathmarcus**
WIP Feat: 
Basic User registration page && Login Page. 
Token is stored on localstorage and into a variable, User info stored onto a variable, both with UserContext. 
There are still some visual issues, mobile responsive still not ok on both pages, font family, submit button is displaced to the bottom when an error appears on an Input field, missing links, need to rename some placeholders so everything is in spanish. Validation schema is missing in Login. 
Functionality is ok "

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Button = (props) => {
+    console.log(props)
```
Comment (alepaez-dev) [inline src/components/Button/Button.jsx:4]: .

File: src/components/schemas/index.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import * as yup from "yup";
+
+const profilePicRules = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/;
```
Comment (alepaez-dev) [inline src/components/schemas/index.jsx:3]: Usualmente los schemas van fuera de la carpeta components.

`src/schemas`

**Review by alepaez-dev — APPROVED** [review]
nice

**Review by Josdandar — APPROVED** [review]
Revisado.

**Review by alfredopizana — APPROVED** [review]

**Pull request merged**